.github/workflows/validate_pr_labels_and_release_notes.yml

```diff
@@ -1,9 +1,13 @@
 name: 🏷️ Validate PR labels and release notes
-
 on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, labeled, unlabeled, edited]
   pull_request:
     types: [opened, synchronize, reopened, labeled, unlabeled, edited]
-
+    
+permissions:
+  pull-requests: write
+  issues: write
```

Comment:
is this needed to access to access `github.rest.issues`?

Reply:
Yes, according to the [github docs](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token), I find it misleading but they mean everything under "issues"